### DISCUSSION TITLE
Contribute Section in Table of Contents

### DISCUSSION THREAD

**Discussion by Darkflame72** [withastro/starlight#1694] [Feature Requests]
### What version of `starlight` are you using?

0.21.0

### What is your idea?

Contribute section as part of the Table of Contents.

Contain the 

- `edit this page` link that is at the bottom of the page
- `link to contribute docs` link to a page or external link with guide to contributing
- `Translate this page` link to details to translate
- `Feedback` link to raise github issues or other external link for feedback on the docs

<img width="191" alt="Screenshot 2024-03-29 at 11 30 31 PM" src="https://github.com/withastro/starlight/assets/31436575/dbc5f243-e400-45dd-aa5a-d8a6e3f60c67">

### Why is this feature necessary?

Often doc sites need contributions to expand them, translations, feedback on the docs, etc similar to what the Astro Docs use. This information should be available to the reader at an easy to see location to encourage them to contribute.

### Do you have examples of this feature in other projects?

- https://github.com/withastro/docs/blob/main/src/components/starlight/TableOfContents.astro

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
We’re actually planning to move away from this pattern in Astro’s docs — see https://github.com/withastro/docs/issues/6845

While it can definitely seem like an attractive bit of the screen to use in desktop layouts, it is not obvious where the content should go on narrower viewports. See the issue linked above for ideas we have about how to handle it better.

**Comment by HiDeoo** [comment]
As this section no longer exists in Astro Docs and the issue linked by Chris has now been addressed, I'm sharing a screenshot of the updated version:

<img width="763" height="222" alt="image" src="https://github.com/user-attachments/assets/36a51ab0-f713-48ea-bb71-88b830705b41" />

The changes were implemented in https://github.com/withastro/docs/pull/8197 if anyone is curious. I don't think we have any short term plans to add such section to Starlight itself as it's very specific to each project and can be easily implemented by users themselves using [component overrides](https://starlight.astro.build/guides/overriding-components/) so I'm going to close this discussion for now.
